src/pages/home/base/mobile-core/lifecycle/behavior-reference.md

```diff
@@ -226,33 +204,27 @@ In this scenario, the `lifecycleStart` call (5) is not detected as a new session
 **Missing start**  
 This scenario shows an example of [consecutive `lifecyclePause` API calls](#consecutive-lifecyclepause-api-calls). The new session is not detected because `lifecycleStart` is not called.
 
-1. Expected Lifecycle event(s)
-2. `lifecyclePause`
-3. *App closed by user*
-4. Optionally: *Session timeout window passed*
-5. *App launched*
-6. `lifecyclePause` (missing `lifecycleStart` call)
-
 ![](./assets/android/lifecycle-missing-start.svg)
 
 <!-- mermaid.js diagram definition
 graph LR
-    A("1.<br/>Expected Lifecycle event(s)")
-    A ==> B(2.<br/><code>lifecyclePause</code>)
-    B ==> C(["3.<br/>App closed by user"])
-    C ==> D(["4.<br/>(Optionally)<br>Session timeout window passed"])
-    D ==> E(["5.<br/>App launched"])
-    E ==> F("6.<br/><code>lifecyclePause</code><br>(Missing <code>lifecycleStart</code> call)")
+    A(["1.<br/><code>App launched</code>"])
+    A ==> B("2.<br/><code>lifecycleStart</code>")
+    B ==> C("3.<br/><code>lifecyclePause</code>")
+    C ==> D(["4.<br/>App closed by user"])
+    D ==> E(["5.<br/>(Optionally)<br>Session timeout window passed"])
+    E ==> F(["6.<br/><code>App launched</code>"])
+    F ==> G("7.<br/><code>lifecyclePause</code><br>(Missing <code>lifecycleStart</code> call)")
 
     classDef dashedPill fill:#d3d3d3,stroke:#000,stroke-dasharray: 5 5,color:#000;
     classDef regularPill fill:#3273de,stroke:#3273de,color:#fff;
     classDef regularBox fill:#009c3b,stroke:#009c3b,color:#fff;
     classDef incorrectBox fill:#EB1000,stroke:#EB1000,color:#fff;
 
-    class A,B regularBox;
-    class C,E regularPill;
-    class D dashedPill;
-    class F incorrectBox;
+    class A,D,F regularPill;
+    class B,C regularBox;
+    class E dashedPill;
+    class G incorrectBox;
 -->
 
 In this scenario, the last `lifecyclePause` call (6) is effectively a [consecutive `lifecyclePause` API call](#consecutive-lifecyclepause-api-calls), with the consequences of:
```

Comment:
Update the reference numbers here. The lifecyclePause calls are at 3 and 7, not 2 and 6.